scripts/just.config.ts

```diff
@@ -96,7 +99,15 @@ export function preset() {
         );
   });
 
-  task('ts', series('ts:compile', 'ts:postprocess'));
+  task('ts', () => {
+    const args = getJustArgv();
+
+    return series(
+      'ts:compile',
+      'ts:postprocess',
+      condition('babel:postprocess', () => !!args.babel),
```

Comment:
In the past we've usually done these conditionals based on the presence of the relevant config, or it could also be done based on the presence of a dependency. Might be worth considering here too.

Reply:
> it could also be done based on the presence of a dependency

Nice idea, love it. Will do 👍

Reply:
(I will go with a config as I would add another Babel plugin that should help us with bundle size)